core/nwb.base.yaml

```diff
@@ -253,12 +267,12 @@ groups:
     dtype: text
     doc: Description of this collection of images.
   datasets:
-  - neurodata_type_inc: Image
-    doc: Images stored in this collection.
+  - neurodata_type_inc: BaseImage
+    doc: Images stored in this collection. Can be either Image or ExternalImage types.
```

Comment:
```suggestion
    doc: Images stored in this collection.
```

Reply:
Same reason as above I guess, no need to commit to early.